src/maps/empiricalPolicy.ts

```diff
@@ -11,11 +12,20 @@ export const mapEmpiricalPolicyResponseToEmpiricalPolicy: MapFunction<EmpiricalP
   })
 }
 
-export const mapEmpiricalPolicyToEmpiricalPolicyResponse: MapFunction<EmpiricalPolicy, EmpiricalPolicyResponse> = function(source: EmpiricalPolicy): EmpiricalPolicyResponse {
+export const mapEmpiricalPolicyToEmpiricalPolicyResponse: MapFunction<EmpiricalPolicy, EmpiricalPolicyResponse> = function(source) {
   return {
     'retries': source.retries,
     'retry_delay': source.retryDelay,
     'max_retries': source.maxRetries,
     'retry_delay_seconds': source.retryDelaySeconds,
   }
+}
+
+export const mapEmpiricalPolicyToEmpiricalPolicyRequest: MapFunction<EmpiricalPolicy, EmpiricalPolicyRequest> = function(source) {
+  return {
+    'retries': source.retries,
+    'retry_delay': source.retryDelay,
```

Comment:
Then this could be like `source.retries ? source.retryDelay : null`

Reply:
interestingly it seems that the API does support sending a `retryDelay` even when `retries` is `null`.. I can't think of a good reason for it to work that way, but I'm tempted to leave as is in case there's some situation where maybe retries is set in code and we just want to set `retryDelay` in the frontend to be something different?

Reply:
Oh dang, good on ya for investigating that. Yeah that's an interesting sort of developer principle to work out.. Like default to flexible/open, or default to some sort of sanitized outcome. Idk

Reply:
Yeah I'd support what the api supports. Map them both individually.